### ISSUE TITLE
[Client] - Change Missed Lines Column for Commit Detail and PR Page Tables

### ISSUE THREAD

**Issue by adrian-codecov** [codecov/engineering-team#605]
For designs, refer to this Figma, https://www.figma.com/file/YayQHDiX9uY7J7MVTtsVFX/Codecov-Light-Plan?type=design&node-id=1-172&mode=design&t=dDUbYM8skOPbTdy5-0

This change affects **ALL** users regardless of tier, and doesn't need to be under a feature flag.

- We want to change the `Missed lines` to `Lines` different convention
- The value is no longer an integer but rather a set of different states:
  - If there are > 0 misses, we say "<#> missing"
  - If there are 0 misses, we say "Fully Covered"
  - * we won't do  partials here
- Affected tables: 
  - Commit detail page 
    - 'files changed' tab table
  - PR page
    - 'files changed' tab table
 

```[tasklist]
## Blocked by:
- [ ] https://github.com/codecov/engineering-team/issues/622
- [ ] https://github.com/codecov/engineering-team/issues/619
```

```[tasklist]
### Tasks
- [ ] https://github.com/codecov/engineering-team/issues/748
```
